vite.config.ts

```diff
@@ -20,6 +20,10 @@ export default defineConfig({
       exclude: ['**/*.test.ts'],
       outDir: 'dist',
       insertTypesEntry: true,
+      copyDtsFiles: true,
```

Comment:
This is the essential fix. The `.d.ts` files are not copied by default, that's why the types are missing.